.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 # most precedence.
 
 # Primary repo maintainers
-*       @faddat @mustafapylons @MikeSofaer @faisalnaveedRNS @kjawadDeveloper2 @Amean
+*       @faddat @mustafapylons @MikeSofaer @faisalnaveedRNS @kjawadDeveloper2 @Amean @afti-githobo
```

Comment:
Kindly remove mustafapylons as the git is complaining about non-existence of the account.